src/frontend/screens/Observation/Buttons.tsx

```diff
@@ -62,6 +100,24 @@ export const ButtonFields = ({
     ]);
   }
 
+  async function handlePressShare() {
+    const base64Urls = attachmentBase64Queries
+      .filter(q => !!q.data)
+      .map(q => q.data!);
+    const {lon, lat} = observation;
```

Comment:
This can lead to images being improperly being shared. If there are alot of photos to share, it could take several seconds for `attachmentBase64Queries` to have `data`. So if the user pressed the share button right away, it would not share any photos because useAttachmentsBase64Query has not completed yet,

We did not have to deal with this in `mapeo-mobile` because urls were retrieved synchronosly.

In order to avoid this, we should block the share button with a loader (UIActivityIndicator) until the query has completed.